src/components/Grants/GrantCard.tsx

```diff
@@ -36,6 +37,12 @@ const GrantCard = ({ grant }: GrantCardProps) => {
   const category: ProposalGrantCategory = configuration.category
   const intl = useIntl()
   const t = useFormatMessage()
+  const isOneTimePayment = ONE_TIME_PAYMENT_TIERS.includes(grant.configuration.tier as ProposalGrantTier)
```

Comment:
I know that this array has only 2 elements but as a general rule, whenever possible, we should use a Set instead of an Array since array.include is O(n) and Set.has is O(1)